PhotoPoints/app/src/main/res/layout/fragment_details.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/fragment_details"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/fragmentBackground">
+
+    <!--
+
+    TODO: Use recycler/scroll view to probide a space for uploaded pictures to appear below the description
+
+    -->
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="370dp"
+        android:layout_height="247dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="20dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:srcCompat="@tools:sample/backgrounds/scenic" />
+
+    <TextView
+        android:id="@+id/text_plantName"
+        android:layout_width="370dp"
+        android:layout_height="27dp"
+        android:layout_marginTop="5dp"
+        android:layout_marginEnd="20dp"
+        android:ems="10"
+        android:text="name_placeholder"
+        android:textSize="20dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/divider" />
+
+    <TextView
+        android:id="@+id/text_scientificName"
+        android:layout_width="370dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="20dp"
+        android:ems="10"
+        android:text="scientific_placeholder"
+        android:textSize="10dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_plantName" />
```

Comment:
Same as above.
Also for the following TextViews. Consider using a style for android elements that are identical. That way you do not have to change it for every control every time.

Reply:
I'm not certain what you mean by style, do you mean the size of the fonts for the scientific name and common name? If so i'm almost certain that having the scientific name in smaller text below the common name is a pretty common implementation of this sort of thing

Reply:
I mean using the styles resource. You can find an example on how to do this here: https://github.com/MobileApps-Cascadia/photopoints-android/pull/49/files (Adding the style in the style-resource file and using that style to apply it to all similar controls.
Let me know if you need help !